packages/webapp/src/components/Table/styles.module.scss

```diff
@@ -14,25 +14,57 @@
  */
 
 .table {
-  padding: 0.5px;
```

Comment:
@Duncan-Brain I had to remove this not to show the table content from the space above the table header when scrolling. Please let me know if this change will break anything.

https://github.com/LiteFarmOrg/LiteFarm/assets/33141219/889fec04-0dbd-4f48-8598-233da58230e0

Reply:
Yeah no worries from me, I just copied it over from previous version!